tests/channel_testing_utils.py

```diff
@@ -1,14 +1,15 @@
-import pathlib
+# Copyright (C) 2022 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
This module has some functions adapted from Powerloader (same license, different copyright holder). I am not sure how this is handled here, but better mention it in case you know?